wireapp/kalium
# chore: move event storage to db

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

as we have few values that been saved in the shared pref, and it quite affect the performance with many operations, we decided to move those value to the metadata table in the DB


### Solutions

insert the last event id in the meta data table


### Testing

#### How to Test

run the app and check if the last event id saved in the meta data table and not in the user shared pref


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
